core/buildTokens.js

```diff
@@ -0,0 +1,147 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Existing Ionic script files are lowercase and use kebab-case if multi-word. Can we rename this file and update the script command?

Reply:
Ah, I actually had it named tokens.js and changed it to buildTokens.js as I saw the setupJest.js followed that approach. Will change it 👍

Reply:
Fixed!